config/locales/en.yml

```diff
@@ -503,6 +505,7 @@ en:
         full_name: 'Full name'
         phone: 'UK telephone number'
         email: 'Email address'
+        email_secondary: 'Secondary email address'
```

Comment:
Is the user going to understand the purpose of the secondary email address - do we need a hint or something similar to explain why we have 2 email fields?